index.d.ts

```diff
@@ -0,0 +1,78 @@
+export enum ZigbeeDeviceAccess {
+    STATE = 1,
```

Comment:
Since access property is just a bitmask it should be refactored into something like this (and this definition is explicit btw):

```typescript
export enum ZigbeeDeviceAccess {
    NONE = 0b000,
    STATE = 0b001,
    WRITE = 0b010,
    READ = 0b100,
}
```

for example in current proposal: ZigbeeDeviceAccess.STATE_SET = 3 which is just `0b001 | 0b010 ` but it should look like this:
`ZigbeeDeviceAccess.STATE | ZigbeeDeviceAccess.WRITE`

Reply:
Makes sense. I'll add the change :)

Reply:
Is there any description we can give for what the individual access flags mean?
I can't really imagine what the difference is between `STATE` and `STAGE_GET`

Reply:
>I can't really imagine what the difference is between STATE and STAGE_GET
Yes, you've already noticed problem with this aliases :)

Check documentation about access property
https://github.com/Koenkk/zigbee2mqtt.io/blob/develop/docs/information/exposes.md#access